apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -70,9 +69,9 @@ export const ExploreProfile: React.FC<ExploreProfileProps & {
                         </div>
                     }
                 </div>
-                {profile.actor.summary &&
+                {profile.bio &&
                     <div
-                        dangerouslySetInnerHTML={{__html: profile.actor.summary}}
+                        dangerouslySetInnerHTML={{__html: profile.bio}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)